### PR TITLE
docs: enrich migration guide

### PR DESCRIPTION
**Pull request by ndr-brt**
PR migrated from https://github.com/catenax-ng/product-edc/pull/818

### CODE REVIEW

File: docs/migration/Version_0.1.x_0.3.x.md [hunk 1]
```diff
@@ -11,14 +13,20 @@ details at the [official documentation on swaggerhub](https://app.swaggerhub.com
 - Removed field `assetId` from `ContractOffer`. It was always null though, so there should be nothing to do about it.
 - on `POST /contractdefinitions` a `duration` field can be added to control the duration of the contract.
 - added the `GET /assets/{id}/address` endpoint to being able to retrieve the stored `DataAddress`
-- concerning the Business Partner Validation Extension a so called orconstraint must be used instead of an IN operator. More details in the [related documentation entry](https://github.com/catenax-ng/product-edc/tree/0.3.0/edc-extensions/business-partner-validation)
 
 ## Settings changes
+- refactored the HTTP server contexts (more details on the [related decision record](https://eclipse-edc.github.io/docs/#/submodule/Connector/docs/developer/decision-records/2022-11-09-api-refactoring/renaming)). They need to be refactored as:
```
Comment (florianrusch-zf) [inline docs/migration/Version_0.1.x_0.3.x.md:18]:
```suggestion

- refactored the HTTP server contexts (more details on the [related decision record](https://eclipse-edc.github.io/docs/#/submodule/Connector/docs/developer/decision-records/2022-11-09-api-refactoring/renaming)). They need to be refactored as:
```

File: docs/migration/Version_0.1.x_0.3.x.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Migration from 0.1.x to 0.3.x
 
+## Deprecation
```
Comment (florianrusch-zf) [inline docs/migration/Version_0.1.x_0.3.x.md:3]:
```suggestion

## Deprecation
```

File: docs/migration/Version_0.1.x_0.3.x.md [hunk 3]
```diff
@@ -11,14 +13,20 @@ details at the [official documentation on swaggerhub](https://app.swaggerhub.com
...
+  - `web.http.validation`, `web.http.controlplane` and `web.http.dataplane` become `web.http.control`
+- Healthcheck api now it's exposed under the `management` context.
 - Removed default value for setting `edc.transfer.proxy.token.verifier.publickey.alias` so it must be valued accordingly
 - made the state machine settings configurable so it will be possible to tune them accordingly. More details in the [related documentation entry](https://eclipse-edc.github.io/docs/#/submodule/Connector/docs/developer/performance-tuning).
-- refactored the HTTP server contexts, they can be configured with the settings group `web.http.management`, `web.http.control`, `web.http.protocol`, `web.http.public`. More details on the [related decision record](https://eclipse-edc.github.io/docs/#/submodule/Connector/docs/developer/decision-records/2022-11-09-api-refactoring/renaming)
+- renamed `edc.receiver.http.endpoint` to `edc.receiver.http.dynamic.endpoint`
 - renamed `edc.oauth.public.key.alias` setting to `edc.oauth.certificate.alias`
 
 ## Other changes:
 - Supported `/public` data plane endpoint without trailing slash, that can be eventually removed from the configuration
```
Comment (florianrusch-zf) [inline docs/migration/Version_0.1.x_0.3.x.md:29]:
```suggestion

- Supported `/public` dataplane endpoint without trailing slash, that can be eventually removed from the configuration
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]
Just found some markdown linting problems

**Review by florianrusch-zf — APPROVED** [review]

**Review by paullatzelsperger — APPROVED** [review]

**Pull request merged**